src/components/utils.js

```diff
@@ -17,3 +17,13 @@ export function buildRegExpFromDelimiters(delimiters) {
   const escapedDelimiterChars = escapeRegExp(delimiterChars);
   return new RegExp(`[${escapedDelimiterChars}]+`);
 }
+
+/**
+ * Returns true when the tag is drag-n-drop enabled
+ * @param {object} props props of the tag element
+ * @returns {boolean} true/false
+ * The three different properties which controls this function are moveTag, readOnly and allowDragDrop.
+ */
+export function canDrag(props) {
```

Comment:
can you rename props to `params` or `data` as the prop word makes sense in react,  for this util its just arguments, then the documentation above will also remove `prop` word twice